_extras/snippets_library/UCL_sge/16/top_output.snip

```diff
@@ -1,26 +1,16 @@
- 7361 root      20   0       0      0      0 S   2.2  0.0  36:53.49 ptlrpcd_35                                                                               
-26386 hallen    20   0 4321956 123520   6740 S   2.2  0.0   0:03.81 conda                                                                                    
-30830 pcerro    20   0  117344   1656   1312 S   2.2  0.0   0:05.70 deployer_ooj.sh                                                                          
-    1 root      20   0  196108   3932   1644 S   0.0  0.0  82:49.29 systemd                                                                                  
-    2 root      20   0       0      0      0 S   0.0  0.0   6:14.69 kthreadd                                                                                 
-    3 root      20   0       0      0      0 S   0.0  0.0   0:06.40 ksoftirqd/0                                                                              
-    5 root       0 -20       0      0      0 S   0.0  0.0   0:00.00 kworker/0:0H                                                                             
-    8 root      rt   0       0      0      0 S   0.0  0.0   0:46.32 migration/0                                                                              
-    9 root      20   0       0      0      0 S   0.0  0.0   0:00.00 rcu_bh                                                                                   
-   10 root      20   0       0      0      0 S   0.0  0.0   0:00.00 rcuob/0                                                                                  
-   11 root      20   0       0      0      0 S   0.0  0.0   0:00.00 rcuob/1                                                                                  
-   12 root      20   0       0      0      0 S   0.0  0.0   0:00.00 rcuob/2                                                                                  
-   13 root      20   0       0      0      0 S   0.0  0.0   0:00.00 rcuob/3                                                                                  
-   14 root      20   0       0      0      0 S   0.0  0.0   0:00.00 rcuob/4                   
+   PID USER      PR  NI    VIRT    RES    SHR S  %CPU %MEM     TIME+ COMMAND
+ 80382 ucabfoz   20   0 5136376   2.7g  41296 R 100.0  1.4  66:00.43 python
+165958 ucjucho   20   0  110864 105572   1788 R 100.0  0.1   0:03.68 magma
+116468 ucfaeef   20   0   25476  11460   2000 R  99.7  0.0  35:35.22 prank
+ 47193 skgtoni   20   0  135588   4596   1224 S   1.7  0.0 909:21.82 htop
+  7885 ucabfoz   20   0  177668  43784   1256 S   1.3  0.0  16:19.25 tmux
+ 75826 ucabfoz   20   0  173724   2780   1112 S   0.7  0.0  77:07.83 top
+ 77103 ucfaeve   20   0   48.9g   1.2g  13552 S   0.7  0.6   1:45.07 java
+165476 cceagil   20   0   69860   3804   1928 R   0.7  0.0   0:00.34 top
+     9 root      20   0       0      0      0 S   0.3  0.0  84:57.93 rcu_sched
```

Comment:
1. Add a line at the end of the file and 
1. change the usernames to something else, we don't want to show to the world what these users were running. Change your user to `user` and chose different names for everyone else except `root`.

Reply:
Done.